static/js/actions/email.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+export const actionCreatorGenerator = (type: string) => (
+  (args: any) => args === undefined ? { type: type } : { type: type, payload: args }
+);
```

Comment:
take a look [here](http://redux.js.org/docs/basics/Actions.html)

I agree it's probs useful elsewhere, just wanted to get feedback on it before I rewrote both of the other action files! I think if you agree that's a good idea I might open an issue and open a follow-up PR to make that change.